kernel_tracing/metric_evaluation/spec.json

```diff
@@ -0,0 +1,26 @@
+{
+  "cases": [
+    {
+      "name": "IPC TX Time",
+      "script": "../bpftrace_scripts/ipc_tx_time.bt",
+      "shell": [
+        "SOF_LOGGING=none TPLG=/lib/firmware/intel/sof-tplg/sof-hda-generic-4ch.tplg ../../test-case/check-playback.sh -l1",
+        "sleep 5"
+      ],
+      "conditions": [
+        "@avg < 500"
+      ]
+    },
+    {
+      "name": "Suspend/Resume Time",
+      "script": "../bpftrace_scripts/suspend_resume_time.bt",
+      "shell": "../../test-case/check-suspend-resume.sh",
+      "conditions": [
+        "@avg[kretprobe:sof_resume, runtime] < 150000",
+        "@avg[kretprobe:sof_resume, system] < 200000",
+        "@avg[kretprobe:sof_suspend, runtime] < 50000",
+        "@avg[kretprobe:sof_suspend, system] < 1000"
```

Comment:
Does `@avg` mean "average"? Averages hide intermittent bugs, they're not very interesting. 

https://bravenewgeek.com/everything-you-know-about-latency-is-wrong/

Reply:
Yes, it means average. However, it's purely a demo (I just made the numbers up), so it doesn't matter much. I figured it'd be better to let those more familiar with SOF figure out the real metrics/conditions to use. I assume we'll end up with some sort of script to generate test cases with device-specific conditions and tests anyways. I've renamed the spec file to `sample_spec.json` to make it more clear that it's just a demo.